src/neptune/attributes/namespace.py

```diff
@@ -99,6 +102,14 @@ def assign(self, value: Union[NamespaceVal, dict, Mapping], *, wait: bool = Fals
             value = NamespaceVal(value)
 
         for k, v in value.value.items():
+            if len(parse_path(k)) == 0:
```

Comment:
It's Python my dear:
```python
>>> if not []:
...     print("It's working!")
...
It's working!
```

Reply:
I know, but I was on the fence on how many people are ok with that syntax :|

Reply:
That's quite common for Python. https://stackoverflow.com/a/53522

Reply:
Done